server/src/domain/system-config/system-config.core.ts

```diff
@@ -248,6 +250,14 @@ export class SystemConfigCore {
       }
     }
 
+    if (!config.ffmpeg.acceptedVideoCodecs.includes(config.ffmpeg.targetVideoCodec)) {
```

Comment:
I added this so that the target video/audio codec is always included in the accepted lists.